### PR TITLE
Task 2 Serverless

### PR DESCRIPTION
**Pull request by kaseopea**
**Task 2.1 Manual deployment**

Manually created S3 Bucket with public access:
S3 URL: [http://photo-wish-manual.s3-website-eu-west-1.amazonaws.com](http://photo-wish-manual.s3-website-eu-west-1.amazonaws.com) (403 Status)

CloudFront URL: [https://dquizaq26n201.cloudfront.net/](https://dquizaq26n201.cloudfront.net/)

**Task 2.2**

- Serverless-finch/serverless-single-page-app plugins are added and configured
- Add necessary npm script(s) to build and deploy your app
- The app can be built and deployed by running single npm script command

CloudFront URL: [https://d1mrxy17t1su7o.cloudfront.net/](https://d1mrxy17t1su7o.cloudfront.net/)
S3 Bucket not accessible directly: 
[http://photo-wish-shop.s3-website-eu-west-1.amazonaws.com/](http://photo-wish-shop.s3-website-eu-west-1.amazonaws.com/)

### CODE REVIEW

**Comment by generalov** [comment]
@kaseopea , CloudFront URL: d2lkblkj7llov4.cloudfront.net is not accessible to me. Could you please double check?

image

**Comment by kaseopea** [comment]
> @kaseopea , CloudFront URL: d2lkblkj7llov4.cloudfront.net is not accessible to me. Could you please double check?
> 
Please, use this one [https://dquizaq26n201.cloudfront.net](https://dquizaq26n201.cloudfront.net).
I've mistaken with url, PR description updated.

**Review by generalov — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -38,7 +49,9 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
-    "@types/jest": "^24.0.0"
+    "@types/jest": "^24.0.0",
+    "serverless-finch": "2.6.0",
+    "serverless-single-page-app-plugin": "file:./deploy/serverless-single-page-app-plugin"
```
[the remark this answers is not in the document]
Comment (kaseopea) [reply package.json:54]: Nice tip! Thanks a lot!

File: package.json [hunk 2]
```diff
@@ -38,7 +49,9 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
-    "@types/jest": "^24.0.0"
+    "@types/jest": "^24.0.0",
+    "serverless-finch": "2.6.0",
```
[the remark this answers is not in the document]
Comment (kaseopea) [reply package.json:54]: Good catch! Added serverless package to package.json

**Pull request merged**